pkg/experiment/sensitivity/executors.go

```diff
@@ -43,6 +43,9 @@ var (
 	// HPKubernetesMemoryResourceFlag indicates amount of memory that HP task can use.
 	HPKubernetesMemoryResourceFlag = conf.NewIntFlag("kubernetes_hp_memory_resource", "Sets memory limit and request for HP workloads on Kubernetes in bytes (default 4GB).", 4000000000)
 
+	// HPKubernetesGuaranteedClassFlag indicates tha HP workload will run as guarateed class.
+	HPKubernetesGuaranteedClassFlag = conf.NewBoolFlag("kubernetes_hp_guaranteed_class", "Sets resources limit equal requests effectivlly running this PoD as \"Guranteed class\" (by default runs as \"Burstable\" ", false)
```

Comment:
@ppalucki - I would appreciate if we had a talk about this description as I do not really understand it.